src/transports/sentry_transport_winhttp.c

```diff
@@ -15,7 +15,8 @@
 
 typedef struct {
     sentry_dsn_t *dsn;
-    wchar_t *user_agent;
+    char *sdk_user_agent;
+    wchar_t *winhttp_user_agent;
```

Comment:
Argh... sorry for spamming, I was commenting in the sole commit view and the PR doesn't pick up.

This is a little too much for my taste. Let's keep the `wchar_t *user_agent` here and convert to `char*` at the only place it is needed: the call site of `sentry__prepare_http_request()`.

Just `sentry__string_from_wstr()` into a variable because it clones, and then make sure to free that string in the `!req` branch right below and again in the `exit` label.